assets/server/static/js/popup/main.js

```diff
@@ -12,6 +12,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+document.addEventListener("DOMContentLoaded", function () {
+  var categorySelect = document.getElementById("category");
+  var reasonInput = document.getElementById("reason");
```

Comment:
Prefer `document.querySelector`, use `const/let` instead of `var`:

```suggestion
  const categorySelect = document.querySelector("#category");
  const reasonInput = document.querySelector("#reason");
```

Reply:
updated in the latest version.